docs/installation/windows.md

```diff
@@ -1,161 +1,227 @@
+Additionally, you will need to ensure that the Python distribution used is built using the same compiler architecture (x86/x64) as the one you are using to build HELICS, as well as the one that was used to build Boost (as mentioned above).
+ZeroMQ will be built using the CMake build process.
+
+![](../img/windows-command-line-install.png)
+
+
+```bash
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="C:\local\helics-v1.0.0"  -DBUILD_PYTHON_INTERFACE=ON -G "Visual Studio 14 2015 Win64" ..
+cmake --build . --config Release --target install
+```
+otherwise they can be set through CMAKE flags
+```bash
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX="C:\local\helics-v1.0.0"  -DBUILD_PYTHON_INTERFACE=ON -G "Visual Studio 14 2015 Win64" -DPYTHON_INCLUDE_DIR=$(python3-config --prefix)\include\python3.6m\ -DPYTHON_LIBRARY=$(python3-config --prefix)\lib\python3.6m\libpython3.6m.dll ..
+cmake --build . --config Release --target install
+```
+
+![](../img/windows-command-line-success.png)
+
+Add the following to the Windows PYTHONPATH environment variable or run the following in the command line.
+
+```bash
+set PYTHONPATH=C:\local\helics-v1.0.2\python;%PYTHONPATH%
+```
+
+If you open a interactive Python session and import helics, you should be able to get the version of `helics` and an output that is similar to the following.
+
+```bash
+$ ipython
+Python 3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 12:04:33)
+Type 'copyright', 'credits' or 'license' for more information
+IPython 6.2.1 -- An enhanced Interactive Python. Type '?' for help.
+
+In [1]: import helics
+
+In [2]: helics.helicsGetVersion()
+Out[2]: '1.0.2 (04-28-18)'
+
+```
+
+![](../img/windows-python-success.png)
+
+## Building HELICS From Source on Windows with MSYS2 ##
+### Overview ###
+This section will layout the setting up of MSYS2 to compile and install HELICS. This guide will describe all the packages and install instructions for a 64bit build.
+
+### Setting up MSYS2 ###
+MSYS2 provides a linux like terminal environment on your Windows system. MSYS2 can be installed from [here](https://www.msys2.org/). Once MSYS2 has been installed start up msys2.exe. Follow first time updates as described on the MSYS2 website. After MSYS2 has been successfully updated Some packages need to be installed in order to configure and build HELICS. The following packages need to be installed:
+- base-devel
+- mingw-w64-x86_64-toolchain
+- git
+- mingw-w64-x86_64-cmake
+- mingw-w64-x86_64-boost
+- mingw-w64-x86_64-qt5 (only if you want to be able to run cmake-gui which this guide recommends.)
+- mingw-w64-x86_64-zeromq
+
+All packages can be installed by typing the following:
+```bash
+$ pacman -S --needed base-devel mingw-w64_x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-boost mingw-w64-x86_64-qt5
+```
+After all this has been done /mingw64/bin must be in the PATH environment variable. If it isn't then it must be added.

```

Comment:
Should add a note about the mingw-64 shell vs the msys shell, and if using the mingw-64 shell this isn't needed.

Reply:
I've edited the file providing a note about the mingw-64 shell.